### DISCUSSION TITLE
Is foundry became slow ?

### DISCUSSION THREAD

**Discussion by panditdhamdhere** [Cyfrin/foundry-full-course-cu#2731] [General]
anyone facing problem ? in my laptop Foundry the blazing fast running very slow! It takes more than 20 seconds to run compile code any suggestions?

**Comment by EngrPips** [comment]
I haven't faced such an issue before, what machine are you running?

**Comment by panditdhamdhere** [comment]
I am on Asus laptop windows 11 -- WSL 
I was using same foundry couple of weeks ago on same machine It was lightning fast. from last couple of days it is very slow. (I did not update anything.)

**Reply by EngrPips** [reply]
I am not sure what could be wrong, Maybe you are working with a larger codebase now.

**Comment by DCVglobalnetwork** [comment]
Hi, I am using ASUS laptop, windows 11 --WSL 
from my experience daily, yes, I can find similar issuses or something. i get an error with import forge-std/Test.sol but there is no issuse at all but still shows an error so what i do is 

 Clear the cache and recompile to see if it helps
```sh
forge clean
```
update 
```sh
foundryup
```
Some libraries or dependencies or If your laptop has limited resources (RAM, CPU)
```sh
forge install
```
yes again just in case

**Comment by EngrPips** [comment]
Thanks for the heads-up @DCVglobalnetwork
